src/photos/targets/public/App.jsx

```diff
@@ -80,7 +80,7 @@ export class App extends Component {
                     styles['--hide-bar']
                   )}
                 >
-                  <h2 className={styles['pho-content-title']}>{album.name}</h2>
```

Comment:
Shouldn't the css rule be removed from the stylus file?

Reply:
No because it's still used in https://github.com/cozy/cozy-drive/blob/master/src/photos/components/Topbar.jsx for when there's a input displayed (for renaming) and there's some style with it.